### ISSUE TITLE
NVDA announces corner cells again when the character is moved by using character reposition controls in Firefox (Maybe not a bug)

### ISSUE THREAD

**Issue by sbates-idrc** [codelearncreate/c2lc-coding-environment#481]
NVDA announces corner cells again when the character is moved by using character reposition controls in Firefox.  This is different than Chrome and Firefox, where moving character from the corner cells towards the blocked direction (trying to move character to right from L8 for example) Chrome doesn’t announce character’s position written on live region again, but Firefox does. Daniel saw the same behaviour with VoiceOver on iPad.

Potential solution: add a ‘bump’ sonification to indicate that the boundary has been hit.

Original Jira issue: https://fluidproject.atlassian.net/browse/C2LC-568
